### ISSUE TITLE
parseTimeOrError: no parse of "2012-01-12T02:09:58"

### ISSUE THREAD

**Issue by philderbeast** [BlockScope/flare-timing#1]
```
$ __shake-build/mask-track "--file=/data/Australia Forbes 2012.comp-input.yaml" --task=6

mask-track: parseTimeOrError: no parse of "2012-01-12T02:09:58"
CallStack (from HasCallStack):
  error, called at libraries/time/lib/Data/Time/Format/Parse.hs:116:12 in time-1.6.0.1:Data.Time.Format.Parse
```

**Comment by philderbeast** [comment]
Comparing pilot `*.kml` files, the one that doesn't parse omits the zulu suffix on the timestamp.

```
<FsInfo time_of_first_point="2012-01-12T02:48:54Z" -- parses
<FsInfo time_of_first_point="2012-01-12T02:09:58" -- doesn't parse
```

**Comment by philderbeast** [comment]
Fixed by 5bf22c00cbc93a2355a937a325334402c2791150.
